# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Feedback
atlas_msgs/AtlasSimInterfaceState state

